pom.xml

```diff
@@ -1147,6 +1147,11 @@
                 <version>${project.version}</version>
                 <type>test-jar</type>
             </dependency>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>blob-strategy-guice</artifactId>
```

Comment:
Guice modules are registered in `server/container/guice/pom.xml` only...

Reply:
I see `blob-s3-guice`, `blob-memory-guice` ... is stilling declare in this pom

Reply:
Then maybe we should move them?

Reply:
I tried to move some `xxx-guice` dependencies to `server/container/guice/pom.xml` only.
But I'm fail, example `org.apache.james.webadmin.integration.memory.vault.MemoryLinshareBlobExportMechanismIntegrationTest` , it used `LinshareGuiceExtension` of `blob-export-guice`. Then we can't move  `blob-export-guice`
So, I will resolve `blob/deduplication-gc` first

Reply:
If you don't succeed, don't insist that is likely something that could be fixed later IMO